src/components/subtitlesettings/subtitlesettings.template.html

```diff
@@ -72,6 +72,13 @@ <h2 class="sectionTitle">
             </select>
         </div>
 
+        <div class="selectContainer">
+            <select is="emby-select" id="selectTextWeight" label="${LabelTextWeight}">
+                <option value="">${Normal}</option>
```

Comment:
I said "alternative" (pick one).
Let's go my way - "no implicit values".
```suggestion
                <option value="normal">${Normal}</option>
```

Reply:
Yokes i think i messed up @dmitrylyzo did this close because i changed the name of the branch?

Reply:
dc6bfc8f364be1d02e0ef7e52e09b6edbbe3b655

Reply:
> Yokes i think i messed up @dmitrylyzo did this close because i changed the name of the branch?

It seems so. When you rename a branch, a new one is created (a new commit reference) and the old one is removed, which closes the PR.

Reply:
ok i reopened it here with your suggested commit https://github.com/jellyfin/jellyfin-web/pull/3632